src/wizard/interactive_build.jl

```diff
@@ -184,6 +184,83 @@ function diff_srcdir(state::WizardState, prefix::Prefix, ur::Runner)
     return false
 end
 
+function bb_add(client, state::WizardState, prefix::Prefix, platform::Platform, jll::AbstractString)
+    if any(dep->getpkg(dep).name == jll, state.dependencies)
+        println(client, "ERROR: Package was already added")
+        return
+    end
+    new_dep = Dependency(jll)
+    try
+        # This will redo some work, but that may be ok
+        setup_dependencies(prefix, getpkg.([state.dependencies; new_dep]), platform)
```

Comment:
This should now be
```suggestion
        concrete_platform = get_concrete_platform(platform;
                                                  preferred_gcc_version = state.preferred_gcc_version,
                                                  preferred_llvm_version = state.preferred_llvm_version,
                                                  compilers = state.compilers)
        setup_dependencies(prefix, getpkg.([state.dependencies; new_dep]), concrete_platform)
```
see #743.